Extensions/BitmapText/bitmaptextruntimeobject-pixi-renderer.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * The PIXI.js renderer for the Bitmap Text runtime object.
+ *
+ * @class BitmapTextRuntimeObjectPixiRenderer
+ * @constructor
+ * @param {gdjs.BitmapTextRuntimeObject} runtimeObject The object to render
+ * @param {gdjs.RuntimeScene} runtimeScene The gdjs.RuntimeScene in which the object is
+ */
+gdjs.BitmapTextRuntimeObjectPixiRenderer = function(
+  runtimeObject,
+  runtimeScene
+) {
+  this._object = runtimeObject;
+
+  const style = new PIXI.TextStyle();
+  style.fontFamily = runtimeScene
+    .getGame()
+    .getFontManager()
+    .getFontFamily(runtimeObject._fontFamily);
+  style.fontSize = runtimeObject._fontSize;
+  style.wordWrap = runtimeObject._wordWrap;
+  style.wordWrapWidth= runtimeObject._wrappingWidth;
+  style.fill = runtimeObject._color;
+
+  const slugFontName =
+    style.fontFamily + '-' + style.fontSize + '-' + style.fill + '-bitmapFont';
+
+  if (!PIXI.BitmapFont.available[slugFontName]) {
+    PIXI.BitmapFont.from(slugFontName, style, {
+      chars: PIXI.BitmapFont.ASCII,
+      textureWidth: 1024,
+    });
+  }
+
+  // Load (or reset) the text
+  if (this._pixiObject === undefined) {
+    this._pixiObject = new PIXI.BitmapText('BitmapText object', {
+      fontName: slugFontName,
+    });
+
+    this._pixiObject.align = runtimeObject._align;
+
+    this.style = style;
+    this.constructorSlugFontName = slugFontName;
+  } else {
+    this.updateColor();
+    this.updateAlignment();
+    this.updateFontFamily();
+    this.updateFontSize();
+    this.updateWordWrap();
+    this.updateWrappingWidth();
+  }
+
+  // update BitmapText with new styles
+  this._pixiObject.updateText();
+
+  runtimeScene
+    .getLayer('')
+    .getRenderer()
+    .addRendererObject(this._pixiObject, runtimeObject.getZOrder());
+
+  // Set the anchor in the center, so that the object rotates around
+  // its center
+  this._pixiObject.anchor.x = 0.5;
+  this._pixiObject.anchor.y = 0.5;
+
+  this.updateTextContent();
+  this.updatePosition();
+  this.updateAngle();
+  this.updateOpacity();
+};
+
+gdjs.BitmapTextRuntimeObjectRenderer = gdjs.BitmapTextRuntimeObjectPixiRenderer;
+
+gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.getRendererObject = function() {
+  return this._pixiObject;
+};
+
+// REVIEW comparé le wrap avec celui de l'IDE
+gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.updateWordWrap = function() {
+  this._pixiObject.style.wordWrap = this._object._wordWrap;
```

Comment:
I've not seen any "wordWrap" like this in the documentation. Did you get confused with the BBText? I can only see a `maxWidth` property. You must set it to 0 if you want to disable word wrap.